README.md

```diff
@@ -93,3 +93,11 @@ yarn start
 ## Stopping Developement Server
 
 To stop the server, use <kbd>Ctrl</kbd> + <kbd>C</kbd>.
+
+## USing DOcker to containerize while working 
+
+To spin up a docker image of the application, use the docker-compose.yml file by running
+```
+docker-compose up --build --force-recreate
```

Comment:
Running this command didn't work for me. Is docker-compose supposed to be a npm package? Do we have to install that separately?

Reply:
Docker for my linux box did not come with it, so I had to apt install docker-compose on my server. I think Windows and Mac versions have a downloadable version. According to this site, the docker compose function should come with the Docker for Desktop installation: https://docs.docker.com/compose/install/

Reply:
I also had to throw the env file as well as do the yarn install command before running the docker-compose command.

Reply:
Should probably add that to your readme instructions.